h2o-py/h2o/automl/autoh2o.py

```diff
@@ -332,6 +334,52 @@ def predict(self, test_data):
             return self._model.predict(test_data)
         print("No model built yet...")
 
+    #---------------------------------------------------------------------------
+    # Download POJO/MOJO with AutoML
+    #---------------------------------------------------------------------------
+
+    def download_pojo(self, path="", get_genmodel_jar=False, genmodel_name=""):
+        """
+        Download the POJO for the leader model in AutoML to the directory specified by path.
+
+        If path is an empty string, then dump the output to screen.
+
+        :param path:  An absolute path to the directory where POJO should be saved.
+        :param get_genmodel_jar: if True, then also download h2o-genmodel.jar and store it in folder ``path``.
+        :param genmodel_name Custom name of genmodel jar
+        :returns: name of the POJO file written.
+        """
+        assert_is_type(path, str)
```

Comment:
same can be accomplished just with

    h2o.download_pojo(self.leader, path, get_jar=get_genmodel_jar, jar_name=genmodel_name)

Reply:
Agree. Will make this change